Doc/howto/clinic.rst

```diff
@@ -1891,3 +1891,82 @@ blocks embedded in Python files look slightly different.  They look like this:
     #[python start generated code]*/
     def foo(): pass
     #/*[python checksum:...]*/
+
+
+How to deprecate positional use of optional parameters
```

Comment:
Only *arguments* can be optional or required. But it does not matter here, because this feature is not limited for optional arguments.

What about the following variant?
```suggestion
How to deprecate passing the arguments positionally
```

Reply:
+1 to what Serhiy says, but I'd drop the "the":

```suggestion
How to deprecate passing arguments positionally
```